spack/packages/sirius/package.py

```diff
@@ -95,7 +95,7 @@ class Sirius(CMakePackage, CudaPackage):
     depends_on('pkgconfig', type='build')
 
     # Python module
-    depends_on('python', when='+python', type=('build', 'run'))
+    depends_on('python', when='+python@3.6:', type=('build', 'run'))
```

Comment:
IMO we should drop this file as well and submit changes directly to the spack package of sirius